orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,19 @@
+topic	url_case_study	url

```

Comment:
no difference
I remember you asked me if I could include more general links, or if it was a special list of url's
the list of url's used earlier belonged to a case study ( i have around 2000 different article links for the same topic)
I added another column with some general links, as suggested
the dataset might not contain just url's , hence I added a column(topics) which does not contain links
it might help for testing